.lintrunner.toml

```diff
@@ -178,11 +178,10 @@ init_command = [
 ]
 
 [[linter]]
-code = 'MYPYNOFOLLOW'
+code = 'MYPYINDUCTOR'
 include_patterns = [
     'torch/_dynamo/**/*.py',
     'torch/_inductor/**/*.py',
-    'torch/_C/_dynamo/**/*.py',
```

Comment:
this doesn't really do anything, it doesn't match the `.pyi` files in that folder, and there are no `.py` files there